src/Sylius/Behat/Context/Api/Shop/CartContext.php

```diff
@@ -117,6 +117,7 @@ public function iAddOfThemToMyCart(int $quantity, ProductInterface $product, str
     }
 
     /**
+     * @Given /^I have ("[^"]+" variant of this product) in the (cart)$/
```

Comment:
I see, that it is not always true, but IMHO steps starting with `given` should be part of the setup context. Preferably, they shouldn't make any requests at all. We already made this mistake in UI contexts, but it is harder there to maintain the same separation.